Apps/fastlane/Fastfile

```diff
@@ -142,6 +132,24 @@ platform :ios do
   end 
 end 
 
+# Functions specific to just the iOS SDK repo can go here: 
+
+lane :iossdk_update_app_version do |arguments| 
```

Comment:
Code that is specific to the iOS SDK got moved out of `ios build` and into this separate function.